config_data.yml

```diff
@@ -42,6 +42,7 @@ inputs:
     dimension_map:
       UNIT_MEASURE: 'Units'
       SERIES: 'Series'
+      REF_AREA|524: ''
```

Comment:
To make this work `REF_AREA` needs to be removed from list below as Brock mentioned

Reply:
Thanks